cnf/build.bnd

```diff
@@ -18,9 +18,23 @@
 		pretty = true; \
 		local = ${build}/release
 
--releaserepo: Release
 -baselinerepo: Release
 
+Bundle-Vendor: FENECON GmbH
```

Comment:
I guess default `Bundle-Vendor` should be 'OpenEMS Association e.V.' what do you think @sfeilmeier?